ui/settingsdlg.ui

```diff
@@ -383,6 +383,179 @@
+               <horstretch>0</horstretch>
+               <verstretch>0</verstretch>
+              </sizepolicy>
+             </property>
+             <property name="title">
+              <string>Additional drive</string>
+             </property>
+             <property name="checkable">
+              <bool>true</bool>
+             </property>
+             <property name="checked">
+              <bool>true</bool>
+             </property>
+             <layout class="QGridLayout" name="gridLayout_18">
+              <item row="5" column="1">
+               <widget class="QLineEdit" name="drive_path"/>
+              </item>
+              <item row="4" column="1">
+               <widget class="QComboBox" name="drive_domain"/>
+              </item>
+              <item row="5" column="0">
+               <widget class="QLabel" name="label_52">
+                <property name="text">
+                 <string>Path:</string>
+                </property>
+               </widget>
+              </item>
+              <item row="3" column="1">
+               <widget class="QComboBox" name="drive_type"/>
+              </item>
+              <item row="4" column="0">
+               <widget class="QLabel" name="label_53">
+                <property name="text">
+                 <string>Backend domain:</string>
+                </property>
+               </widget>
+              </item>
+              <item row="3" column="0">
+               <widget class="QLabel" name="label_51">
+                <property name="text">
+                 <string>Type:</string>
+                </property>
+               </widget>
+              </item>
+              <item row="5" column="2">
+               <widget class="QToolButton" name="drive_path_button">
+                <property name="text">
+                 <string>...</string>
+                </property>
+               </widget>
+              </item>
+              <item row="6" column="0" colspan="3">
+               <widget class="QLabel" name="drive_running_warning">
+                <property name="enabled">
+                 <bool>true</bool>
+                </property>
+                <property name="font">
+                 <font>
+                  <weight>75</weight>
+                  <bold>true</bold>
+                 </font>
+                </property>
+                <property name="styleSheet">
+                 <string notr="true">color: rgb(255, 0, 0);</string>
+                </property>
+                <property name="text">
+                 <string>New drive will be used only at next VM startup</string>
+                </property>
+                <property name="textFormat">
+                 <enum>Qt::AutoText</enum>
+                </property>
+               </widget>
+              </item>
+             </layout>
+            </widget>
+           </item>
+          </layout>
+         </item>
```

Comment:
This looks to revert "Remove obsolete sections from VM settings" commit, at least partially - here is "Additional drive" section..